clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -1715,6 +1715,37 @@ def BitPopcountOp
   }];
 }
 
+def BitLzcntOp : CIR_Op<"bit.lzcnt", [Pure, SameOperandsAndResultType]> {
+  let summary = "Get the number of leading 0-bits in the input";
+  let description = [{
+    Compute the number of leading 0-bits in the input.
+
+    The input integer must be an unsigned integer. The `cir.bit.lzcnt` operation
+    returns the number of consecutive 0-bits at the most significant bit
+    position in the input.
+
+    If the input value is 0, the return value is the size of the input value.
```

Comment:
Add some more description about the differences between the `cir.bit.clz` operation.

Reply:
Yea, that'd be great